ext/integrate/D1037.tex

```diff
@@ -97,7 +97,7 @@
 % \input{lib-intro}
 % \input{utilities}
 \input{iterators}
-\input{ranges-lib}
+% \input{ranges-lib}
```

Comment:
That was an oversight. I just added my ranges-lib.tex. Please rebase and revert this change.